### ISSUE TITLE
can i modify the rotation?

### ISSUE THREAD

**Issue by syo277** [timhul/ClassicSim#333]
I wanna cast adrenaline rush and blade flurry while holy strength buff is on. Is it possible?

**Comment by veluria** [comment]
Yes, this is possible, but currently only by modifying the XML files.
